BankChain/app/src/androidTest/java/nl/tudelft/ewi/ds/bankchain/bank/bunq/BunqTest.java

```diff
@@ -144,4 +153,100 @@ public void testExceptionConfirmationNotFuture() {
 
         assertTrue(bank.confirmException(thr) == thr);
     }
+
+    @Test
+    public void testGetUsers() throws ExecutionException, InterruptedException {
+        CompletableFuture<Void> future;
```

Comment:
Unused, please remove. Please think about putting other variables in BeforeClass or Before methods to remove a lot of boilerplate.